evap/staff/tests/test_views.py

```diff
@@ -2389,6 +2389,26 @@ def test_copy_participants(self):
             self.evaluation.participants.count(), original_participant_count + self.evaluation2.participants.count()
         )
 
+    def test_copy_invalid_participants(self):
+
+        old_evaluation = baker.make(
+            Evaluation,
+            course__semester=self.evaluation2.course.semester,
+            participants=self.evaluation2.participants.all(),
+        )
+        page = self.app.get(self.url, user=self.manager)
+        old_pk = old_evaluation.pk
+        old_evaluation.delete()
+
+        form = page.forms["participant-copy-form"]
+        form["pc-evaluation"] = old_pk
+        res = submit_with_modal(page, form, name="operation", value="copy-participants", status=200)
```

Comment:
I think other code typically uses `page` or `response`, I don't think we've ever used `res` in the code base. For consistency, I'd prefer to use one of the existing variable names.